src/utils/membership.js

```diff
@@ -18,6 +12,30 @@ export const isItemUpdateAllowedForUser = ({ memberships, memberId }) =>
     ),
   );
 
+export const getBestPermissionForMemberFromMemberships = ({
```

Comment:
Maybe just add a description of what is `best`? From what I see here quickly, it's by length?